src/util.h

```diff
@@ -9,9 +9,18 @@
 #include <glib.h>
 #include <gio/gio.h>
 #include <tss2/tss2_tpm2_types.h>
+#if defined(__FreeBSD__)
+#include <sys/endian.h>
+#endif
 
 #include "control-message.h"
 
+#if defined(__FreeBSD__)
+#ifndef POLLRDHUP
```

Comment:
This is only used in the TCTI implementation. Only place that includes `poll.h` at least. This belongs over there.

Reply:
Right, I will move it there.

Reply:
thx